content/800-data-platform/300-data-proxy.mdx

```diff
@@ -239,6 +240,18 @@ Each Data Proxy instance has its own connection pool. The size of each of these
 postgresql://user:password@host:5432/mydb?connection_limit=60
 ```
 
+### Idle connections to the database
+
+The Data Proxy creates several connection pools that maintain open idle connections even when no queries run from your application. The connection pools and any remaining idle connections close after 6 minutes of inactivity (no queries are sent to the database).
+
+In each of the supported locations, the Data Proxy architecture includes multiple clusters that maintain idle connections to the database to ensure high availability in case of failures in any of the clusters.
+
+This architecture has a number of benefits.
+
+- **High availability**. By maintaining a separate connection pool from each cluster in a location, the Data Proxy guarantees that at least one connection pool will be always available to handle queries.
+- **Load balancing**. The Data Proxy distributes the load of the database queries among the number of available connection pools.
+- **Scalability**. New connection pools start when an application experiences peak activity and existing connection pools stop (but two always remain with at least one being active in a separate cluster) when the peak activity subsides.
+
 ## Error codes
 
 Because Data Proxy connections operate differently from a local database connection, error codes will be different from regular ones. In general, all Data Proxy-related errors will start with `P5xxx`.
```

Comment:
I think we should make the mentions of Data Proxy connections and local database connections both plural or both singular. So either:

"Because Data Proxy connections operate differently from local database connections..."

or

"Because a Data Proxy connection operates differently from a local database connection..."